docker/labels.go

```diff
@@ -11,10 +11,11 @@ type Label string
 
 // Libcompose default labels.
 const (
-	NAME    = Label("io.docker.compose.name")
-	PROJECT = Label("io.docker.compose.project")
-	SERVICE = Label("io.docker.compose.service")
-	HASH    = Label("io.docker.compose.config-hash")
+	NAME     = Label("io.docker.compose.name")
+	PROJECT  = Label("io.docker.compose.project")
+	SERVICE  = Label("io.docker.compose.service")
+	HASH     = Label("io.docker.compose.config-hash")
+	RECREATE = Label("io.docker.compose.recreate")
```

Comment:
@dnephin 100% agreed, we will rip out the labels.  I think there is a bit of a confusion in that libcompose already supports `--no-recreate` and `--force-recreate` in this PR.  The problem was that it additionally supported the same behavior from labels.  So we will rip out the labels.  We will ensure that libcompose is a strict subset of docker-compose going forward.  The PR was actually created way back and has been dangling for awhile, so I think it wasn't clear how strictly we would conform to docker-compose at that time.  You've made you point loud and clear, so we will always ensure that we don't add new behavior.  Thanks!